mongo/tests/test_integration.py

```diff
@@ -69,7 +69,8 @@ def test_integration_replicaset_primary_in_shard(instance_integration, aggregato
         'fsynclock',
     ]
     _assert_metrics(aggregator, metrics_categories, replica_tags)
-
+    # Lag metrics are tagged with the state of the member and not with the current one.
+    _assert_metrics(aggregator, ['replset-lag-from-primary-in-shard'])
```

Comment:
What is `replset-lag-from-primary-in-shard`? Is that different than the new metric you add?

Reply:
This is not a metric, this is a metric_category, i.e metrics are loaded from the json file called `metrics-replset-lag-from-primary-in-shard.json`